spim_registration/timelapse/define_czi.bsh

```diff
@@ -14,180 +14,50 @@ print("Threads: "+Prefs.getThreads()); // prints thread setting in output
-		channel_string.append(channel_part);
-		channel_string.append(" ");
-		}
-System.out.println( "Channel String = " + channel_string );
 
-System.out.println("---------------------------------------------------------");
-System.out.println("Illumination Settings:");
-
-// Illuminatin settings
-// This parses illumination and determines the number of illuminations
-// then uses the tokens of illuminations to assemble the string for 
-// illum_string via StringBuilder
-illumination = System.getProperty( "illumination" );
-System.out.println( "Illumination = " + illumination );
-
-// Parses string illumination
-String delims = "[,]";
-String[] illum_token = illumination.split(delims);
-
-// Builds string using the number of tokens and inserts the tokens in the string
-StringBuilder illum_string = new StringBuilder();
-String illum_part;
-
-	for ( int illum= 0; illum< illum_token.length; illum++) {
-		int num_illum = illum + 1;
-		illum_part = "_______illumination_" + num_illum + "=" + illum_token[illum];
-		illum_string.append(illum_part);
-		illum_string.append(" ");
-		}
-
-System.out.println( illum_string );
 
+// Directory, and first .czi
 System.out.println("---------------------------------------------------------");
-System.out.println("Rotation setting:");
-
-rotation_around = System.getProperty( "rotation_around" );
-System.out.println( "rotation_around = " + rotation_around );
 
-// Calibaration
-System.out.println("=========================================================");
-System.out.println("Calibration:");
-manual_calibration_czi = System.getProperty( "manual_calibration_czi" );
-float czi_pixel_distance_x = Float.parseFloat( System.getProperty( "czi_pixel_distance_x" ) );
-float czi_pixel_distance_y = Float.parseFloat( System.getProperty( "czi_pixel_distance_y" ) );
-float czi_pixel_distance_z = Float.parseFloat( System.getProperty( "czi_pixel_distance_z" ) );
-czi_pixel_unit = System.getProperty( "czi_pixel_unit" );
-
-String modify_calibration;
-String manual_calibration_string;
-
-// builds string for calibration override
-if (manual_calibration_czi.equalsIgnoreCase( "No" ) )
-{
-	modify_calibration = "";
-	manual_calibration_string = "";
-	System.out.println( "Calibration set to automatic" );
-}
-
-else if (manual_calibration_czi.equalsIgnoreCase( "Yes" ) )
-{
-	
-	modify_calibration = "modify_calibration ";
-	manual_calibration_string = "pixel_distance_x=" + czi_pixel_distance_x + " " +
-				    "pixel_distance_y=" + czi_pixel_distance_y + " " +
-				    "pixel_distance_z=" + czi_pixel_distance_z + " " +
-				    "pixel_unit=" + czi_pixel_unit + " ";
-	System.out.println( "Calibration set to manual" );
-	System.out.println( "modify_calibration:" + modify_calibration );
-	System.out.println( "manual_calibration_string:" + manual_calibration_string );
+apply_rotation_to_dataset = System.getProperty("apply_rotation_to_dataset");
+apply_rotation_to_dataset_str = "";
+if ( apply_rotation_to_dataset.equalsIgnoreCase( "Yes" ) ) {
```

Comment:
Maybe I am missing something but it looks like that information on the `No` or bad setting has disappeared. Is this intentional?

Reply:
Thats right - another value then "Yes" causes empty apply_rotation_to_dataset_str

Reply:
Sure, however this way the user won't be notified that there is an invalid setting, for example.

Reply:
OK, that makes sense.